test/e2e/testing-manifests/auth/encrypt/run-e2e.sh

```diff
@@ -91,7 +91,7 @@ create_cluster_and_run_test() {
     TEST_ARGS=""
     if [ "${SKIP_RUN_TESTS:-}" != "true" ]; then
         # (--use-built-binaries) use the kubectl, e2e.test, and ginkgo binaries built during --build as opposed to from a GCS release tarball
-        TEST_ARGS="--test=ginkgo -- --v=5 --focus-regex=\[Conformance\] --skip-regex=\[Serial\] --parallel 20 --use-built-binaries"
+        TEST_ARGS="--test=ginkgo -- --focus-regex=\[Conformance\] --skip-regex=\[Serial\] --parallel 20 --use-built-binaries"
```

Comment:
This change is already part of https://github.com/kubernetes/kubernetes/pull/123251